pypy/interpreter/baseobjspace.py

```diff
@@ -1239,10 +1240,12 @@ def call_valuestack(self, w_func, nargs, frame, methodcall=False):
 
             if isinstance(w_func, Function):
                 return w_func.funccall_valuestack(
-                        nargs, frame, methodcall=methodcall)
+                        nargs, frame, methodcall=methodcall, dropvalues=dropvalues)
+
```

Comment:
Here and elsewhere: dropvalues is not really a kwarg

Reply:
I don't understand, does that cause a problem?